upgrade.json

```diff
@@ -26,7 +26,8 @@
   "23": {
     "slave": ["20160516-mbs-8838.sql",
               "20160516-mbs-8720.sql",
-              "20141226-geodesic-index.sql"],
-    "standalone": []
+              "20141226-geodesic-index.sql",
+              "20160310-mbs-4501-alternative-tracklists.sql"],
+    "standalone": ["20160310-mbs-4501-alternative-tracklists-fks.sql"]
```

Comment:
When we're done with the schema changes, we'll run `./admin/CompileSchemaScripts.pl 23` to generate all these individual scripts concatenated together, but all within a single transaction, so that they're not half-applied.

You can see examples in admin/sql/updates/schema-change/. There are different files for slave, slave_only, and standalone. When people run `upgrade.sh` on their local MBS, it'll execute these files depending on their `REPLICATION_TYPE`.

Foreign keys aren't created on slave servers because they'll usually cause replication packets to fail to apply. The state of the database ends up the same, but it violates those constraints sometimes when it's applying individual changes.